.gitignore

```diff
@@ -1,6 +1,9 @@
 .DS_Store
 /tmp
 
+# RubyMine IDE
```

Comment:
nit:  this could be generated by any Jetbrains (or derived) product

Reply:
fixed comment in b6ecae4cfab106309864f5ad8ccd5801220f913d